### ISSUE TITLE
Kotlin tag

### ISSUE THREAD

**Issue by kqvanity** [emmanueltouzery/code-compass.nvim#2]
I'm sure this plugin was created to combat JVM's awful LSP, so I was wondering if Kotlin would be supported as well.

**Comment by emmanueltouzery** [comment]
yes, the plan is to support languages which have poor LSP setup. i didn't have particularly kotlin in mind, but since you mention, i'll put it at the front of the list :+1: 

i first have at least one more thing i'd like to improve with the java support, but then when i have some free time, i'll try to peek at kotlin, and i'll update status on this bug.

**Comment by bqv** [comment]
Second vote for kotlin :+1:

**Comment by emmanueltouzery** [comment]
Dropping since kotlin is getting a proper lsp now
